### DISCUSSION TITLE
After updating Vaultwarden 1.24 to 1.25 my pre-start script won't let Bitwarden startup.

### DISCUSSION THREAD

**Discussion by BobWs** [dani-garcia/vaultwarden#2552] [Q&A] [answered]
After updating Vaultwarden 1.24 to 1.25 my pre-start script won't let Bitwarden startup.

```
while ! curl mariadb:3306 --output /dev/null -0 --connect-timeout 2 --max-time 2 ; do sleep 10 ; done
```

Before 1.25 I was using this script to delay the startup of Vaultwarden untill the Mariadb was running, but after the update to 1.25 Vaultwarden won't start if the script is active. If I deactived the script than Vaultwarden starts just fine.

Any thoughts how to fix this?

**Comment by BlackDex** [comment, marked as answer]
It looks like it doesn't work for me at all, not even on 1.24.0.
Could be a curl version difference maybe.

However, i would suggest to use this instead: https://github.com/dani-garcia/vaultwarden/blob/main/.env.template#L141-L143

**Reply by BobWs** [reply]
Thanks for replying! I will give it a try and let you know.

**Reply by BobWs** [reply]
No more startup problems! Guess it is fixed.
Thanks for your help. 😉
